packages/http/src/validator/validators/body.ts

```diff
@@ -28,11 +28,11 @@ export function deserializeFormBody(
       const encoding = encodings.find(enc => enc.property === property);
 
       if (encoding && encoding.style) {
-        const deserializer = body.get(encoding.style);
-        if (deserializer && schema.properties) {
-          const propertySchema = schema.properties[property];
-          deserialized[property] = deserializer.deserialize(property, decodedUriParams, propertySchema as JSONSchema);
-        }
+        const deserializer = body[encoding.style];
+        const propertySchema = schema.properties?.[property];
+
+        if (propertySchema && typeof propertySchema !== 'boolean')
```

Comment:
Does this fix the difference between JSONSchema versions? AFAIK one of them can have schema equal to bool false.

Reply:
The type system claims this is the definition of `propertySchema`:


![image](https://user-images.githubusercontent.com/1416224/93909863-e4774500-fcc5-11ea-88a2-ab20fc16e948.png)

with such check, I am excluding it's a boolean so it's at least a JSONSchema (the version is yet to determine)

Reply:
👍